docs/nuget-org/package-sponsorship-on-nuget-org.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Package sponsorship on NuGet.org
+description: Learn how to add sponsorship links to your NuGet packages and support package maintainers through NuGet.org's sponsorship feature.
+author: pranathibora14
+ms.author: prabora
+ms.date: 10/15/2025
+ms.topic: conceptual
+ai-usage: ai-generated
+---
+
+# Package sponsorship on NuGet.org
+
+Package maintainers are the foundation of the NuGet.org ecosystem, building and maintaining the packages that developers rely on. The NuGet.org sponsorship feature makes it easier for package consumers to recognize and support the authors behind their favorite packages.
+
+## Overview
+NuGet.org now allows package authors to add sponsorship URLs to their packages. These links appear when the "Sponsor" button on the package details page is clicked.
+
+For package consumers, supporting your favorite packages is now easier than ever. Look for the sponsor icon on NuGet.org to see which packages need support while browsing or managing dependencies.
+
+## Setting up sponsorship for package publishers
+
+### Prerequisites
+
+- You must be the owner or co-owner of a package on NuGet.org
+- Your sponsorship link platform must be from the approved list:
+  - GitHub Sponsors
+  - Patreon
+  - Open Collective
+  - Ko-fi
+  - Tidelift
+  - Liberapay
+
+### Navigate to your package management page
+
+1. Go to [NuGet.org](https://nuget.org) and sign in to your account.
+2. Select your username in the top right corner.
+3. Select **Manage Packages** from the dropdown menu.
+4. Find the package you want to add sponsorship information for and select the edit button.
+
+### Access sponsorship settings
+
+1. On your package management page, scroll down to find the **Sponsorship Links** section.
+2. Select to expand the collapsible **Sponsorship Links** section.
+3. You'll see a form where you can add sponsorship URLs.
+
+   ![Screenshot of the manage package page with the Sponsorship Links section](media/sponsorship-section-manage-package-page.png)
```

Comment:
Hm... will Newtonsoft.Json actually use this feature? Or is it a sample? We may want to use a more sterile/fake example for something like sponsorship.